src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -344,12 +361,6 @@ void vm_RequestNodeCentered(object sender, EventArgs e)
             if (!Double.IsNaN(node.Height))
                 dropPt.Y -= (node.Height / 2.0);
 
-            if (!Double.IsNaN(node.Width))
-                dropPt.X -= (node.Height / 2.0);
-
-            if (!Double.IsNaN(node.Height))
-                dropPt.Y -= (node.Height / 2.0);
-
```

Comment:
just duplicated code, dragged from the search library node is placed not exactly in the center because of that